### PR TITLE
docs: Add general metrics documentation

### LINKED ISSUE #7157: docs: add general metrics documentation

### PR DESCRIPTION
**Pull request by GabyCT**
This PR adds a general metrics introduction documentation for the kata CI.

Fixes #7157

### CODE REVIEW

File: tests/metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+# Kata Containers metrics
+
```
[the remark this answers is not in the document]
Comment (GabyCT) [reply tests/metrics/README.md:2]: @chavafg changes applied

**Review by chavafg — APPROVED** [review]

**Pull request merged**